src/LinuxTracingIntegrationTests/LinuxTracingIntegrationTest.cpp

```diff
@@ -480,6 +480,10 @@ void VerifyOrderOfAllEvents(const std::vector<orbit_grpc_protos::ProducerCapture
         EXPECT_GE(event.module_update_event().timestamp_ns(), previous_event_timestamp_ns);
         previous_event_timestamp_ns = event.module_update_event().timestamp_ns();
         break;
+      case orbit_grpc_protos::ProducerCaptureEvent::kCgroupMemoryUsage:
+        UNREACHABLE();
```

Comment:
Thanks for the suggestion! As we test the memory events in `MemoryTracingIntegrationTest`, so I'm thinking that it would be fine if we ignore the memory events here, WDYT?

Reply:
Oh I was not aware of this. Yeah sounds good. You can add a comment here, though.